DataFormats/METReco/src/classes_def.xml

```diff
@@ -186,7 +190,9 @@
   </class>
   <class name="edm::Wrapper<reco::CSCHaloData>"/>
 
-  <class name="reco::GlobalHaloData" ClassVersion="10">
+  <class name="reco::GlobalHaloData" ClassVersion="12">
+   <version ClassVersion="12" checksum="2525701804"/>
+   <version ClassVersion="11" checksum="1191211594"/>
```

Comment:
as above, classversion 11 with checksum 1191211594 can be removed and the one with 2525701804 set to have 11